site/usage/compile.md

```diff
@@ -20,10 +20,16 @@ If you want access to the compiled Vega spec from a Javascript program, you can
 var vgSpec = vl.compile(vlSpec).spec;
 ```
 
-By default, warnings and errors are printed to the JavaScript console. To customize how errors and warnings are handled, you can pass a customize logger to the compile function. 
+### Customized Configuration
```

Comment:
It isn't very clear that compile has  two parameters the second one is `opt` and Opt can have config/logging.  

https://vega.github.io/vega/docs/api/view/#view is one way to do it.

Reply:
You're right. I improved it.